kymatio/scattering3d/backend/torch_backend.py

```diff
@@ -58,11 +58,129 @@ def compute_integrals(input_array, integral_powers):
     return integrals
 
 
+class Pad(object):
+    def __init__(self, pad_size, input_size, pre_pad=False):
+        """Padding which allows to simultaneously pad in a reflection fashion
+        and map to complex. 
+
+        Parameters
+        ----------
+        pad_size : list of 4 integers
+            Size of padding to apply [top, bottom, left, right].
+        input_size : list of 2 integers
+            Size of the original signal [height, width].
+        pre_pad : boolean, optional
+            If set to true, then there is no padding one simply adds the
```

Comment:
this is not great semantically. do we have this everywhere?

Reply:
lets make an issue.

Reply:
What is going to be the merge order of these PRs? Will this be merged before or after the one removing `pre_pad`

Reply:
since we have the `prepad` refactor PR, this will need to be removed here, because we are merging this after